Code/LauncherUnified/launcher_generator.cmake

```diff
@@ -6,6 +6,9 @@
 #
 #
 
+# This will be used to turn on "PerMonitor" DPI scaling support. (Currently there is no way in CMake to specify "PerMonitorV2")
+set(O3DE_DPI_AWARENESS "PerMonitor")
```

Comment:
should this be a cache variable to make it easier for users to set?
```suggestion
# This will be used to turn on "PerMonitor" DPI scaling support. (Currently there is no way in CMake to specify "PerMonitorV2")
set(O3DE_DPI_AWARENESS "PerMonitor" CACHE STRING "Set the Launcher's VS_DPI_AWARE state.  Valid values are PerMonitor, ON, or OFF.")
```

Reply:
I'm inclined not to do this for now, because it's a bit misleading. Our NativeWindow_Windows code is actually _expecting_ this flag to be turned on, and the flag being off was causing some bugs. I would suggest waiting until someone finds the need to fix up the NativeWindow_Windows code to detect the awareness setting and handle both cases correctly before exposing this as a user setting.